SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -331,26 +336,39 @@ int64 USpatialActorChannel::ReplicateActor()
 	{
 		FOutBunch DummyOutBunch;
 
-		FClassInfo* ClassInfo = NetDriver->TypebindingManager->FindClassInfoByClass(Actor->GetClass());
-
 		for (UActorComponent* ActorComponent : Actor->GetReplicatedComponents())
 		{
-			if (ClassInfo->SubobjectClasses.Contains(ActorComponent->GetClass()))
+			FUnrealObjectRef ObjectRef = NetDriver->PackageMap->GetUnrealObjectRefFromObject(ActorComponent);
+
+			if (ObjectRef != SpatialConstants::NULL_OBJECT_REF)
 			{
-				bWroteSomethingImportant |= ReplicateSubobject(ActorComponent, RepFlags);
+				FClassInfo& SubobjectInfo = *Info->SubobjectInfo[ObjectRef.Offset];
+
+				bWroteSomethingImportant |= ReplicateSubobject(ActorComponent, &SubobjectInfo, RepFlags);
 				bWroteSomethingImportant |= ActorComponent->ReplicateSubobjects(this, &DummyOutBunch, &RepFlags);
 			}
 		}
 
-		for (UObject* Subobject : NetDriver->TypebindingManager->GetHandoverSubobjects(Actor))
+		if (HandoverShadowDataMap.Num() > 0)
```

Comment:
This is always true because `HandoverShadowDataMap` will always contain the entry for the actor itself. Maybe that shouldn't be the case since not all actors have handover properties.